lib/inferno/dsl/fhir_evaluator/evaluator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module FhirEvaluator
+  class Evaluator
+    def initialize(the_ig)
+      @ig = the_ig
+    end
+
+    def evaluate(data, summary = nil, config = Config.new)
+      summary ||= DataSummary.new(data)
+      context = EvaluationContext.new(@ig, data, summary, config)
+
+      active_rules = []
+      config.data['Rule'].each_key do |rulename|
```

Comment:
You're using the key and the value here, so you should use `.each do |rule_name, whatever_you_call_the_value|` rather than `each_key`.

Reply:
Addressed